x/evm/client/cli/query.go

```diff
@@ -572,26 +610,43 @@ func queryCtTransferPayload(cmd *cobra.Command, args []string) error {
 		toAmountHi,
 		remainingBalance,
 		transferProto.DecryptableBalance,
-		proofs)
-	if err != nil {
-		return err
+		proofs,
 	}
-	return queryClientCtx.PrintString(hex.EncodeToString(bz))
-}
 
-func getCtAccount(queryClient cttypes.QueryClient, address, denom string) (*cttypes.Account, error) {
-	ctAccount, err := queryClient.GetCtAccount(context.Background(), &cttypes.GetCtAccountRequest{
-		Address: address,
-		Denom:   denom,
-	})
-	if err != nil {
-		return nil, err
+	if len(auditors) > 0 {
+		var evmAuditors []cttypes.EvmAuditor
+		auditorsProto := transferProto.Auditors
+		for _, auditorProto := range auditorsProto {
+			encryptedTransferAmountLo, _ := auditorProto.EncryptedTransferAmountLo.Marshal()
+			encryptedTransferAmountHi, _ := auditorProto.EncryptedTransferAmountHi.Marshal()
+			transferAmountLoValidityProof, _ := auditorProto.TransferAmountLoValidityProof.Marshal()
+			transferAmountHiValidityProof, _ := auditorProto.TransferAmountHiValidityProof.Marshal()
+			transferAmountLoEqualityProof, _ := auditorProto.TransferAmountLoEqualityProof.Marshal()
+			transferAmountHiEqualityProof, _ := auditorProto.TransferAmountHiEqualityProof.Marshal()
+			// check if the auditor address does not exist in the map
+			if auditorEvmAddress, exists := seiToEvmAddressMap[auditorProto.AuditorAddress]; !exists {
```

Comment:
In what scenario would this happen? Isn't every auditor in the list `auditors` from the list of auditors passed in?

Reply:
I think I added this check when there was a bug and the other part of the code, to make it clear why failure in map lookup occurred. And kept it since. Should not happen ideally